### PR TITLE
Setup pep8 check

### PR DESCRIPTION
**Pull request by trasher**
In order to avoid beginners (like me :p) issues; add a Github actions config to check pep8 on commits.

This PR is based on #2; I'll rebase once it gets merged.

### CODE REVIEW

File: custom_components/ecodevices/ecodevicesapi/__init__.py [hunk 1]
```diff
@@ -19,29 +19,48 @@ def host(self):
...
+                r.request.url,
+                content,
             )
 
     def ping(self) -> bool:
         try:
             self._request({"cmd": 10})
             return True
-        except:
+        except Exception:
```
Comment (trasher) [inline custom_components/ecodevices/ecodevicesapi/__init__.py:50]: I'm not entirely sure for this one; my python is "rusty".

Comment (Aohzan) [reply custom_components/ecodevices/ecodevicesapi/__init__.py:50]: i'm not sure, I used this : https://wiki.python.org/moin/HandlingExceptions
```import sys
try:
  untrusted.execute()
except: # catch *all* exceptions
  e = sys.exc_info()[0]
  write_to_page( "<p>Error: %s</p>" % e )```

**Comment by Aohzan** [comment]
I format the rewrite with black (https://developers.home-assistant.io/docs/development_guidelines/), do you want to add the github action to the branch `rewrite` ?

**Comment by trasher** [comment]
see  #5

File: custom_components/ecodevices/ecodevicesapi/__init__.py (hunk 1 above)
Comment (trasher) [reply custom_components/ecodevices/ecodevicesapi/__init__.py:50]: OK, so I guess the best way would have been to exclude that one for checks; or to do differently (seem it is what you did on your rewrite branch).

**Pull request closed without merging**